tests/hikari/impl/test_rest.py

```diff
@@ -872,9 +978,13 @@ class StubResponse:
         rest_client._parse_ratelimits = mock.AsyncMock()
         rest_client._handle_error_response = mock.AsyncMock(side_effect=exit_exception)
         rest_client._acquire_client_session = mock.Mock(return_value=mock_session)
-        with mock.patch.object(asyncio, "gather", new=mock.AsyncMock()):
-            with pytest.raises(exit_exception):
-                await rest_client._request(route)
+        rest_client._stringify_http_message = mock.Mock()
+        with mock.patch.object(rest, "_LOGGER", new=mock.Mock(getEffectiveLevel=mock.Mock(return_value=100))) as logger:
+            with mock.patch.object(asyncio, "gather", new=mock.AsyncMock()):
+                with pytest.raises(exit_exception):
+                    await rest_client._request(route)
+
+        assert logger.log.call_count == 0
```

Comment:
shouldn't you be testing the response, not logging?

Logging may change without notice, or be reworded. Ideally we shouldn't be testing that as the main test case, but I would assume you want to check the response of `rest_client._request` to ensure that it returns the right stuff in this case, right?

Reply:
This test does that, the logging is just random testing for an extra case. I have no idea why i didnt make a separate test for it (which im going to do now)